releases.md

```diff
@@ -45,49 +45,49 @@ As of 2019-03-29, PELUX 3.1 has been released! Documentation can be found
 
 |Sources for all the images| Commit hash |
 | ------------------------ | ----------- |
-| [Source release](https://pelux.io/artifacts/pelux/3.1/sources/source-release/) | 6266e8bcb8247d54b32119eefc72e93687b693be |
+| [Source release](https://pelux.io/artifacts/pelux/4.0/sources/source-release/) | 6266e8bcb8247d54b4.019eefc72e93687b693be |
 
 #### Images
 
 | Hardware Platform | Variant | Size |
 | ----------------- | ------- | ---- |
-| Intel x86-64      | [Minimal](https://pelux.io/artifacts/pelux/3.1/binaries/intel/core-image-pelux-minimal-dev-intel-corei7-64.wic) <br> [Qt Auto Neptune](https://pelux.io/artifacts/pelux/3.1/binaries/intel/core-image-pelux-qtauto-neptune-dev-intel-corei7-64.wic) | 3.9 GB <br> 5.4 GB |
-| ARP               | [Minimal](https://pelux.io/artifacts/pelux/3.1/binaries/arp/core-image-pelux-minimal-dev-arp.wic) <br> [Qt Auto Neptune](https://pelux.io/artifacts/pelux/3.1/binaries/arp/core-image-pelux-qtauto-neptune-dev-arp.wic)  | 3.9 GB <br> 5.4 GB |
-| Raspberry Pi 3    | [Minimal](https://pelux.io/artifacts/pelux/3.1/binaries/rpi3/core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg) <br> [Qt Auto Neptune](https://pelux.io/artifacts/pelux/3.1/binaries/rpi3/core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg) (with known limitations) | 1.1 GB <br> 1.6 GB |
-| Qemu              | [Minimal](https://pelux.io/artifacts/pelux/3.1/binaries/qemu/core-image-pelux-minimal-dev-qemux86-64.ext4) | 1.1 GB |
+| Intel x86-64      | [Minimal](https://pelux.io/artifacts/pelux/4.0/binaries/intel/core-image-pelux-minimal-dev-intel-corei7-64.wic) <br> [Qt Auto Neptune](https://pelux.io/artifacts/pelux/4.0/binaries/intel/core-image-pelux-qtauto-neptune-dev-intel-corei7-64.wic) | 3.9 GB <br> 5.4 GB |
```

Comment:
Sizes are wrong.

Reply:
Wait, this is in progress

Reply:
Also, we want the .wic.bz2 and .wic.bmap files in this table as well? Or perhaps we should remove the .wic files and only have .wic.bz2?

I also wonder if we should put the checksums here. Checksums are not easily discoverable in previous releases at least.

Reply:
Could perhaps just make the commit hash a link to the commit in the repo...?